.github/workflows/triage-move-unlabelled.yml

```diff
@@ -3,14 +3,15 @@ name: Move unlabelled from needs info columns to triaged
 on:
   issues:
     types: [unlabeled]
-    
+
 jobs:
   Move_Unabeled_Issue_On_Project_Board:
     name: Move no longer X-Needs-Info issues to Triaged
     runs-on: ubuntu-latest
+    # Skip in forks
     if: >
-        ${{
-        !contains(github.event.issue.labels.*.name, 'X-Needs-Info') }}
+      github.repository == 'vector-im/element-android' &&
+      !contains(github.event.issue.labels.*.name, 'X-Needs-Info')
```

Comment:
@kittykat do you know why we had `${{` and `}}` here before?

Reply:
There are a few different ways to write `if` expressions. Using `>` and no curly brackets is the most universally reliable one so this change is fine.